src/api/graphhopper.d.ts

```diff
@@ -54,6 +54,7 @@ export interface RoutingProfile {
 }
 
 export interface Path extends BasePath {
+    // todo: do we really need the geojson dependency? it was included in mapbox but its not included in OL
```

Comment:
do we?

Reply:
Ok I was more thinking about not using it at all. But let's just keep it for now.